homeassistant/components/tplink/strings.json

```diff
@@ -24,5 +24,41 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "current_consumption": {
+        "name": "Current consumption"
+      },
+      "total_consumption": {
+        "name": "Total consumption"
```

Comment:
Do we not have these already somewhere else ?

Reply:
3 other places right now

Reply:
(https://docs.google.com/spreadsheets/d/12KKos-HhRS5nMBokVNX-Scy2n_RqZcGF-T3qLRMsC6I/edit#gid=1068986889)